app/src/main/java/com/einmalfel/podlisten/DownloadReceiver.java

```diff
@@ -297,6 +297,8 @@ public void onReceive(Context context, Intent intent) {
                                 intent.getLongExtra(DownloadManager.EXTRA_DOWNLOAD_ID, 0L));
           updateDownloadQueue(context, false);
           break;
+        default:
+          break;
```

Comment:
We don't expect any other values here, so IMHO it's better to throw AssertionError instead of "break;"